src/blockchains/erc20/erc20_types.ts

```diff
@@ -10,5 +10,7 @@ export type ERC20Transfer = {
   logIndex: number
   type?: string,
   primaryKey?: number
+  fromBalance?: string,
```

Comment:
The idea is to provide also those two fields to the Transfer data and let Flink use it